homeassistant/components/vultr/sensor.py

```diff
@@ -109,10 +109,13 @@ def name(self):
     @property
     def native_value(self):
         """Return the value of this given sensor type."""
+        value = self.data.get(self.entity_description.key)
+        if value == "not a number":
```

Comment:
I think this is not a real value, but something that has been used in tests.

According to the API specification of Vultr

![image](https://user-images.githubusercontent.com/195327/211602600-81829d72-09c7-44f4-9f28-01faf7f1db9c.png)

Therefore, I think it is not right to filter the state here.